### PR TITLE
Add v1.24.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
**Todos:**

- [x] Finish content
- [x] Update image links
- [x] Update contributors list

### CODE REVIEW

**Comment by andrea-p** [comment]
schedule(11/01/2017 6:00am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **11/01/2017 at 6:00amundefined (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/384#issuecomment-340880336)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

File: posts/2017-11-01-serverless-v1.24.0.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework_v1.24.jpg
+authors:
+  - PhilippMuns
+---
+
+<img align="right" src="https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework_v1.24.jpg" width="210px" >
+
+Today we're proud to announce Serverless Framework v1.24.
+
+v1.24 is a special release—it's the first one which was entirely run by our recently-onboarded Serverless Framework v1 maintainers!
```
Comment (ac360) [inline posts/2017-11-01-serverless-v1.24.0.md]: Maybe a bit more clarity on maintainers:

v1.24 is a special release.  We recently made a few of the Serverless Framework's major open-source contributors official maintainers and this is the first release which was run entirely by them.

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**